fedimint-server/src/net/peers.rs

```diff
@@ -32,26 +31,15 @@ use crate::metrics::{PEER_CONNECT_COUNT, PEER_DISCONNECT_COUNT, PEER_MESSAGES_CO
 use crate::net::connect::{AnyConnector, SharedAnyConnector};
 use crate::net::framed::AnyFramedTransport;
 
-/// Owned [`Connector`](crate::net::connect::Connector) trait object used by
-/// [`ReconnectPeerConnections`]
-pub type PeerConnector<M> = AnyConnector<PeerMessage<M>>;
-
-/// Connection manager that automatically reconnects to peers
-///
-/// `ReconnectPeerConnections` is based on a
-/// [`Connector`](crate::net::connect::Connector) object which is used to open
-/// [`FramedTransport`](crate::net::framed::FramedTransport) connections. For
-/// production deployments the `Connector` has to ensure that connections are
-/// authenticated and encrypted.
```

Comment:
Do we still describe the architecture of the network stack somewhere? If so, let's link there, if not why remove?

Reply:
I did not find the description to helpful but maybe its just me.

Reply:
Can bring it back if you want?